File: Final Project/Food_Ordering_Application/Food_Ordering_Application/Model/Restaurant.cs
namespace Food_Ordering_Application.Model
{
    public class Restaurant
    {
       
            public int RestaurantId { get; set; }
            public string? Name { get; set; }
            public string? Address { get; set; }
            public string? PhoneNumber { get; set; }
            public double Rating { get; set; }
            public string? CuisineType { get; set; }

            public string? Image {  get; set; }

           
            public bool IsActive { get; set; } // To track if the restaurant is open or closed
            public int OwnerId { get; set; } // Foreign key to User (RestaurantOwner)

            public User? Owner { get; set; }
            public ICollection<MenuItem>? MenuItems { get; set; }
            public ICollection<Order>? Orders { get; set; }
        }

    }

File: Final Project/Food Order Application/Angular/src/app/Restaurant.ts
   
export interface Restaurant{
   
  
    restaurantId:number,
    name: string,
    address: string,
    phoneNumber:string,
    rating: number,
    cuisineType:string,
    image: string,
    isActive: boolean,
    ownerId: number,
    owner?:any[],
    menuItems?: any[],
    orders?: any[]
  }